src/main.ts

```diff
@@ -20,7 +20,9 @@ async function main() {
 
     let endpoint: IAuthorizer = await AuthorizerFactory.getAuthorizer();
     var taskParams = TaskParameters.getTaskParams(endpoint);
-    let credential: TokenCredential = new DefaultAzureCredential()
+    let credential: TokenCredential = new DefaultAzureCredential();
+    const revisionName = `${taskParams.containerAppName}--${taskParams.revisionNameSuffix}`;
```

Comment:
Why is there a `--`? Reference docs or comments would be appreciated.
Can we put `--` in revisionNameSuffix?

Reply:
This is documentation describing the format.
https://learn.microsoft.com/en-us/azure/container-apps/revisions#revision-name-suffix
![image](https://user-images.githubusercontent.com/4566555/191149408-e137e6bf-8246-412c-9b76-198308aed700.png)

Reply:
Would you like to comment on this URL link about this magic symbol? I think we should know this specification.

Reply:
Added.